.github/workflows/deploy.yml

```diff
@@ -12,20 +12,16 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: '3.7'
-      - name: configure node
-        uses: actions/setup-node@v1
-        with:
-            node-version: '10.x'
       - name: install
         run: |
           pip install -e .[all]
-          npm install -g eslint
           sudo apt-get install -y graphviz
       - name: build
         run: |
           make html
       - name: deploy
         uses: peaceiris/actions-gh-pages@v3
```

Comment:
If it's just to deploy to GitHub pages, would you like to try this one? 

https://github.com/meitar/jekyll-builder-for-github-pages-action

I'm using it for my blog: https://github.com/kinow/kinoshita.eti.br/blob/master/.github/workflows/main.yml

The `ACCESS_TOKEN` is an application key on my account, with permission set to `public_repo`.

![image](https://user-images.githubusercontent.com/304786/76899826-25e4fd00-68fd-11ea-85ea-805ee9de4074.png)

I remember trying the two top actions that I found on GitHub Actions market place, but they refused to work no matter how many times I tried. Then started looking on StackOverflow and blogs, and found this action with almost no stars, but from a developer that I had seen somewhere else (either some other project or conference).

Reply:
> The ACCESS_TOKEN is an application key on my account, with permission set to public_repo.

We've been down this road before - https://github.com/metomi/rose/pull/2222

Basically personal access tokens have a really scary scope, they are personal, there are no organisation access tokens.

For these reasons we have backed out of auto-deployment in the past. Deploy keys - being repo-specific give us a way forward!